src/foundry/foundry.js/textureUtils.d.ts

```diff
@@ -27,8 +23,8 @@ declare function loadTexture(
   }?: {
     /**
      * A fallback texture to use if the requested source is unavailable or invalid
-     * @defaultValue `null`
+     * @defaultValue `undefined`
      */
     fallback?: string;
```

Comment:
I think it only makes sense to document this in one place, not 2. Also, if there is no explicit default value in foundry, we typically do not document it.